File: application/controllers/anggota.php
<?php

class anggota extends CI_Controller{

    public function __construct()
    {
        parent::__construct();
        $this->load->model('anggota_model');
    }
    public function index(){

        $data['anggota']= $this->anggota_model->get_all();
        $this->load->view('templates/header');
        $this->load->view('templates/sidebar');
        $this->load->view('templates/topbar');
        $this->load->view('anggota/index', $data);
        $this->load->view('templates/footer');
    }
    public function tambah()
    {
        $this->load->view('templates/header');
        $this->load->view('templates/sidebar');
        $this->load->view('templates/topbar');
        $this->load->view('anggota/tambah');
        $this->load->view('templates/footer');
    }

    // ======================
    // SIMPAN
    // ======================
    public function simpan()
    {
        $this->load->library('form_validation');
        $this->form_validation->set_rules('nomor_anggota', 'nomor_anggota', 'required');
        $this->form_validation->set_rules('nama_anggota', 'nama_anggota', 'required');
        $this->form_validation->set_rules('alamat', 'alamat', 'required');
        $this->form_validation->set_rules('telepon', 'telepon', 'required');
        $this->form_validation->set_rules('email', 'email', 'required');
        $this->form_validation->set_rules('tanggal_daftar', 'tanggal_daftar', 'required');
        $this->form_validation->set_rules('status', 'status', 'required');
        if($this->form_validation->run()==FALSE){
            $this->tambah();
        } else{
            $data=[
                'nomor_anggota' => $this->input->post('nomor_anggota'),
                'nama_anggota' => $this->input->post('nama_anggota'),
                'alamat' => $this->input->post('alamat'),
                'telepon ' => $this->input->post('telepon'),
                'email' => $this->input->post('email'),
                'tanggal_daftar' => $this->input->post('tanggal_daftar'),
                'status' => $this->input->post('status')
            ];
            $this->anggota_model->insert($data);

            $this->session->set_flashdata('success', 'Data Berhasil Disimpan');
            redirect('anggota');
        }
    }

     public function hapus($no)
    {
        $this->anggota_model->delete($no);
        $this->session->set_flashdata('success', 'Data Berhasil dihapus');

        redirect('anggota');
    }
    

    public function edit($no)
    {
        $data['anggota']= $this->anggota_model->get_by_id($no);
        $this->load->view('templates/header');
        $this->load->view('templates/sidebar');
        $this->load->view('templates/topbar');
        $this->load->view('anggota/edit', $data);
        $this->load->view('templates/footer');
    }
    public function update($no)
    {
        $this->load->library('form_validation');
        $this->form_validation->set_rules('nomor_anggota','nomor_nggota', 'required');
        $this->form_validation->set_rules('nama_anggota','nama_anggota', 'required');
        $this->form_validation->set_rules('alamat','alamat', 'required');
        $this->form_validation->set_rules('telepon','telepon', 'required');
        $this->form_validation->set_rules('email','email', 'required');
        $this->form_validation->set_rules('status','status', 'required');
        if($this->form_validation->run()==FALSE){

        }else{
            $data=[
                'nomor_anggota' => $this->input->post('nomor_anggota'),
                'nama_anggota' => $this->input->post('nama_anggota'),
                'alamat' => $this->input->post('alamat'),
                'telepon ' => $this->input->post('telepon'),
                'email' => $this->input->post('email'),
                'tanggal_daftar' => $this->input->post('tanggal_daftar'),
                'status' => $this->input->post('status')
            ];
            $this->anggota_model->update($no, $data);
            $this->session->set_flashdata('success','Data Berhasil di update');
            redirect('anggota');
        }
    }
}

File: database/anggota.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Text;
namespace Db_ci
{
    #region Anggota
    public class Anggota
    {
        #region Member Variables
        protected int _no;
        protected string _nomor_anggota;
        protected string _nama_anggota;
        protected string _alamat;
        protected string _telepon;
        protected string _email;
        protected unknown _tanggal_daftar;
        protected unknown _status;
        #endregion
        #region Constructors
        public Anggota() { }
        public Anggota(string nomor_anggota, string nama_anggota, string alamat, string telepon, string email, unknown tanggal_daftar, unknown status)
        {
            this._nomor_anggota=nomor_anggota;
            this._nama_anggota=nama_anggota;
            this._alamat=alamat;
            this._telepon=telepon;
            this._email=email;
            this._tanggal_daftar=tanggal_daftar;
            this._status=status;
        }
        #endregion
        #region Public Properties
        public virtual int No
        {
            get {return _no;}
            set {_no=value;}
        }
        public virtual string Nomor_anggota
        {
            get {return _nomor_anggota;}
            set {_nomor_anggota=value;}
        }
        public virtual string Nama_anggota
        {
            get {return _nama_anggota;}
            set {_nama_anggota=value;}
        }
        public virtual string Alamat
        {
            get {return _alamat;}
            set {_alamat=value;}
        }
        public virtual string Telepon
        {
            get {return _telepon;}
            set {_telepon=value;}
        }
        public virtual string Email
        {
            get {return _email;}
            set {_email=value;}
        }
        public virtual unknown Tanggal_daftar
        {
            get {return _tanggal_daftar;}
            set {_tanggal_daftar=value;}
        }
        public virtual unknown Status
        {
            get {return _status;}
            set {_status=value;}
        }
        #endregion
    }
    #endregion
}